lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/templates/deployment.yaml

```diff
@@ -25,6 +30,15 @@ spec:
               mountPath: {{ .Values.lighty.workdir }}/{{ .Values.lighty.configDirectoryName }}
             - name: secrets-volume
               mountPath: {{ .Values.lighty.workdir }}/{{ .Values.lighty.restconf.keyStoreDirectory }}
+          {{- if .Values.deploymentSecurity.useContainerSecurity }}
+          securityContext:
+            allowPrivilegeEscalation: true
```

Comment:
Setting allowPrivilegeEscalation=false is strongly recommended with this strategy.

https://kubernetes.io/docs/concepts/policy/pod-security-policy/

Reply:
Fixed